README.md

```diff
@@ -1,19 +1,19 @@
-# Elrond Node Deploy Scripts - Mainnet
+# MultiversX Node Deploy Scripts
 
 ## INTRODUCTION
 
 The current scripts version aims to bring the validator experience to a higher standard.
-This variant of the scripts can be used on any of the Elronds Networks (mainnet,testnet or devnet) by setting the new `ENVIRONMENT` variable.
-The `ENVIRONMENT` variable will direct the scripts to use the correct elrond-configs repositories (elrond-config-mainnet|testnet|devnet).
-Running an observers plus proxy stack is also supported on all Elrond Networks.
+This variant of the scripts can be used on any of the MultiversX Networks (mainnet,testnet or devnet) by setting the new `ENVIRONMENT` variable.
+The `ENVIRONMENT` variable will direct the scripts to use the correct mx-chain-configs repositories (mx-chain-mainnet|testnet|devnet-config).
```

Comment:
this is confusing. please use `mx-chain-mainnet-config, mx-chain-testnet-config, or mx-chain-devnet-config`

Reply:
Done !